src/components/NewComps/GiddhFor/index.js

```
@@ -1,24 +1,25 @@
-import data from "./data.json";
 import { useState } from "react";
 import style from "./GiddhFor.module.scss";
+import Image from "next/image";
 
-export default function GiddhFor() {
+export default function GiddhFor({ compdata }) {
+  if (compdata?.content?.length === 0) return null;
   const [activeTab, setActiveTab] = useState(0);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)